javascripts/discourse/components/popular-tags.js

```diff
@@ -36,6 +36,10 @@ export default class PopularTags extends Component {
     }
   }
 
+  get viewAllUrl() {
+    return `/tags`;
+  }
+
```

Comment:
What's the reason for making this a getter rather than inlining `"/tags" in the template`? Overridability?

Reply:
No big reason... consistency with the other component that does the same thing (top contributors), I guess?